### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -5,108 +5,108 @@
...
-    <string name="add_hide_files">Add Hide Folder</string>
-    <string name="folder_selector_hints">Hints for folder selector: \n\t 1. Click \'Use This Folder\' to select the Folder\n\t 2. Always select the file from the local storage \n\t(You can slide from the right of the screen to open local files)</string>
-    <string name="remove_hide_path">Remove Hide Path?</string>
-    <string name="remove_hide_path_description">Are you sure to remove path \'%s\' from hiding?</string>
+    <string name="show_debug_info">Debug info</string>
+    <string name="debug_info_description">Shows info to help fix problems.</string>
+    <string name="invalid_dsm_provider">DSM: Invalid DSM provider. Have you installed and activated the DSM provider on your device?</string>
+    <string name="hide_app_failed_admin_inactive">Could not hide app: Administration not active.</string>
+    <string name="add_hide_files">Add hidden folder</string>
+    <string name="folder_selector_hints">Hints for folder selector: \n\t 1. Click \"Use This Folder\" to select the Folder\n\t 2. Always select a file on the device \n\t(Slide from the right of the screen to open local files)</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:44]: There is no "Use This Folder" string…

Comment (deltazefiro) [reply app/src/main/res/values/strings.xml:44]: Ah, It's a button in the android filepicker provided by the system:

**Comment by deltazefiro** [comment]
thx for the work! Some strings look weird in settings:

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -5,108 +5,108 @@
     <string name="hide">Hide</string>
     <string name="visible_status">Visible</string>
     <string name="hidden_status">Hidden</string>
-    <string name="visible_moto" translatable="false">"\"Night has fallen. Be careful.\" "</string>
-    <string name="hidden_moto" translatable="false">"\"Secrets never emerge under the sun.\" "</string>
-    <string name="app_about" translatable="false">\nAmarok %s\n\n
```
Comment (deltazefiro) [inline app/src/main/res/values/strings.xml:10]: I'd rather mark them as not translatable. They are for decoration only :D

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -5,108 +5,108 @@
...
     <string name="security">Security</string>
-    <string name="panic_button">Panic Button</string>
-    <string name="panic_button_description">Display an Emergency-Hide floating button over other apps. The button only shows in unhide state.</string>
-    <string name="in_app_update_failed">New update available but failed to fetch. Please update manually.</string>
-    <string name="failed_to_open_doc_tree">Failed to open system file selector</string>
-    <string name="failed_to_open_doc_tree_description">Oops! Seems that your system does not have a default filemanager? Please long press the \"Add Hide Folder\" button and set path manually.</string>
-</resources>
\ No newline at end of file
+    <string name="panic_button">Panic button</string>
+    <string name="panic_button_description">Floating emergency button used to hide apps shown over all other apps.\nOnly shows when apps are unhidden.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:108]:
```suggestion
    <string name="panic_button_description">Floating emergency button used to hide apps again.\nIt is shown over all other apps.</string>
```

**Comment by deltazefiro** [comment]
Thx again! 
But I found some translations change the original meanings. Perhaps that's because I don't provide a screenshot! 
Oh I can push to this patch branch... I'll try to fix them myself later.
The CI seems broken - not your fault - I should make it work first :disappointed:

**Comment by comradekingu** [comment]
@deltazefiro 
That is what I think too. If you explain (and maybe do screenshots) I can try to improve the ones that don't work.

**Comment by deltazefiro** [comment]
> If you explain (and maybe do screenshots) I can try to improve the ones that don't work.

Thank you. I made some improvements by myself.
I've fixed and kicked off the auto build CI — you can download the app from [here](https://github.com/deltazefiro/Amarok-Hider/suites/10873195950/artifacts/548542546) (or if you can't open the link, get it from "Show all checks -> Details -> Summary -> artifact")
Would you mind installing it and having a look?

**Pull request merged**